rerun_bridge/src/rerun_bridge/rerun_ros_interface.cpp

```diff
@@ -132,15 +128,14 @@ void log_image(
         }
         rec.log(
             entity_path,
-            rerun::DepthImage(
-                {static_cast<size_t>(img.rows), static_cast<size_t>(img.cols)},
-                rerun::TensorBuffer::f32(img)
-            )
-                .with_meter(1.0)
+            rerun::DepthImage(rerun::Collection<float>(img), width_height(img)).with_meter(1.0)
         );
     } else {
         cv::Mat img = cv_bridge::toCvCopy(msg, "rgb8")->image;
-        rec.log(entity_path, rerun::Image(tensor_shape(img), rerun::TensorBuffer::u8(img)));
+        rec.log(
+            entity_path,
+            rerun::Image::from_rgb24(rerun::Collection<uint8_t>(img), width_height(img))
```

Comment:
Seems related to this issue:
* https://github.com/rerun-io/rerun/issues/7081

But I think this code this is fine, since it borrows data that lives longer than `rerun::Image`